### DISCUSSION TITLE
Required tech skills to contribute to Terminus development?

### DISCUSSION THREAD

**Discussion by clarkwang** [Eugeny/tabby#3497] [General]
I'm very interested in the project though I know very little about web related technology. Could someone give me some idea where I can start from?

**Comment by Eugeny** [comment]
Being an Electron app, Terminus is basically a website. The https://github.com/Eugeny/terminus/blob/master/HACKING.md guide explains how to set up you dev environment and start making changes.

To work on it effectively, you need to have basic understanding of [Typescript](https://www.typescriptlang.org/) and [Angular](https://angular.io/) - from that point on it's just the usual web dev.
